ENVIRONMENT.md

```diff
@@ -0,0 +1,24 @@
+
+| Name                                      | Description                                                                                                                                                                                                              |            Range/Type            |  Required?   | Default |
+| ----------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | :------------------------------: | :----------: | :-----: |
+| `API_PORT`                                | HTTP port that the application listens on                                                                                                                                                                                |           1025 - 65535           |              |  3000   |
+| `BLOCKCHAIN_SCAN_INTERVAL_MINUTES`        | How many minutes to delay between successive scans of the chain for new accounts (after end of chain is reached)                                                                                                         |               > 0                |              |   180   |
+| `CAPACITY_LIMIT`                          | Maximum amount of provider capacity this app is allowed to use (per epoch) type: 'percentage' 'amount' value: number (may be percentage, ie '80', or absolute amount of capacity)                                        | JSON [(example)](./env.template) |      Y       |         |
+| `CONNECTIONS_PER_PROVIDER_RESPONSE_PAGE`  | Number of connection/page to request when requesting provider connections from webhook                                                                                                                                   |               > 0                |              |   100   |
+| `FREQUENCY_URL`                           | Blockchain node address                                                                                                                                                                                                  |      http(s): or ws(s): URL      |      Y       |         |
+| `GRAPH_ENVIRONMENT_DEV_CONFIG`            | [NOTE]: Only used for Dev environments. Add the graph environment config in JSON format only used for Dev environments. Be careful to escape any inner quotes as this is in a .env file and will be parsed by the shell. | JSON [(example)](./env.template) | Y (if 'Dev') |         |
+| `GRAPH_ENVIRONMENT_TYPE`                  | Graph environment type. This can be 'Dev' or 'Rococo' or 'Mainnet'.                                                                                                                                                      |       Dev\|Rococo\|Mainnet       |      Y       |         |
```

Comment:
Is there something we need to do to support `Paseo`?

Reply:
Paseo would probably use the mainnet config for now, actually--the schema ids on paseo match mainnet, due to how they were deployed.

Reply:
I'll file a ticket in `graph-sdk` to put in a Paseo config.

Reply:
nvm; `graph-sdk` already has a config for `TestnetPaseo`. I'll update the docs to include that option.